cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,147 @@
+# Log Level Extension
+
+## Abstract
+This extension defines attributes that MAY be included within a CloudEvent
+to describe the "level" or "importance" of an event in relation to other events
+produced by a unique event source.
+
+Often systems produce events in form of logs, and these types of events usually share
+this "log-level" property. This extension aims to provide a standard way for
+describing this property in a language agnostic form
+
+## Attributes
+
+### loglevelname 
+
+- Type: `String`
+- Description: Human readable representation of the log level
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a non-empty string
+  - SHOULD be lowercase
+  - RECOMMENDED values include `emergency`, `alert`, `critical`, `error`, 
+    `warning`, `notice`, `info`, `debug`, and `verbose`, but  others MAY be used.
+  - SHOULD have a one-to-one relationship with a distinct `loglevelnum` in the scope of
+   the `source`
+
+### loglevelnum 
+
+- Type: `Integer`
+- Description: A numerical representation of the level of importance of the event. 
+  Conforms to the severity ordering specified by 
+  [RFC5424](https://www.rfc-editor.org/rfc/rfc5424.html#section-6.2.1): severity of
+  all levels is assumed to be numerically ascending from most important to least
+  important.
+  This specification does not define "importance" or meaning of each value other than
+  an event with a lower numerical value MUST be of more importance than an event with
+  a higher numerical value in the same `source`.
+
+- Constraints
+  - OPTIONAL
+  - if present, MUST NOT be negative
+  - RECOMMENDED values include
+    - `0` for `emergency`
+    - `1` for `alert`
+    - `2` for `critical`
+    - `3` for `error`
+    - `4` for `warning`
+    - `5` for `notice`
+    - `6` for `info`
+    - `7` for `debug`
+    - `8` for `verbose`
+
+## RECOMMENDED Log Mappings
+
+### Syslog
+
+All syslog log records SHOULD assign the severity level to `loglevelnum`. 
+In addition to that The corresponding `loglevelname`s to each of the severity values
+
+| Syslog Severity | `loglevelname` | `loglevelnum` |
+| --------------- | -------------- | ------------- |
+| 0               | `emergency`    | 0             |
+| 1               | `alert`        | 1             |
+| 2               | `critical`     | 2             |
+| 3               | `error`        | 3             |
+| 4               | `warning`      | 4             |
+| 5               | `notice`       | 5             |
+| 6               | `info`         | 6             |
+| 7               | `debug`        | 7             |
```

Comment:
verbose?

Reply:
Sadly, there is no `verbose` in syslog.
The verbose recommended values exists for other systems which usually have a level below `debug`

Reply:
Right - I'd misread things, apologies. Feel free to resolve.